docs/integrations/observability/_index.rst

```diff
@@ -0,0 +1,184 @@
+
+   .. code:: bash
+
+      kubectl create ns det-monitoring
+
+-  Change directory to the determined repo ``tools/observability``. This can be done with
+
+   .. code:: bash
+
+      git clone https://github.com/determined-ai/determined.git && \
+        cd determined/tools/observability
+
+***************
+ Token refresh
+***************
+
+Determined Prometheus export endpoint is secured by authentication. As a result a Determined
+authentication token is needed for the Prometheus scraper. Determined tokens have an expiration of 1
+week. So we are going to configure a token refresh cronjob to run on the Kubernetes cluster.
+
+#. Create a Determined account that will be used in the job.
+
+.. code:: bash
+
+   det -u admin user create tokenrefresher
+
+2. Change the password of the Determined account.
+
+.. code:: bash
+
+   det -u admin user change-password tokenrefresher
+
+3. Store the username and password inside a credential.
+
+.. code:: bash
+
+   kubectl -n det-monitoring create secret generic token-refresh-username-pass \
+     --from-literal="creds=tokenrefresher:testPassword1"
+
+4. Create the job and cronjob.
+
+.. attention::
+
+   ``tokenRefresher.yaml`` won't work in every possible Kuberenetes set up. If you have a unique use
+   case it might need to be modified to work. Hardcoding the Determined master ip can reduce a lot
+   of assumptions the script makes.
+
+.. code:: bash
+
+   kubectl -n det-monitoring apply -f tokenRefresher.yaml
+
+5. Wait for a few minutes then check the ``det-prom-token`` secret was created and ``det-token`` is
+   more than 0 bytes.
+
+.. code:: bash
+
+   kubectl -n det-monitoring describe secret det-prom-token
+
+***********************
+ Install DCGM Exporter
+***********************
+
+The DCGM exporter is used to allow Prometheus to get GPU metrics. This can be installed in a variety
+of different ways. If you are deploying in a cloud based environment you should follow their
+documentation.
+
+A setup method for GKE is included here for convenience. For other clouds or on prem deployments you
+may have to install the DCGM exporter slightly differently and change the
+``additionalScrapeConfigs`` accordingly in the ``grafana-prom-values.yaml`` in later steps.
+
+If you are deploying on prem it is recommended to reference the `Nvidia docs on installing the DCGM
+exporter
+<https://docs.nvidia.com/datacenter/cloud-native/gpu-telemetry/latest/kube-prometheus.html#setting-up-dcgm>`__.
+
+#. Create a namespace for the exporter
+
+.. code:: bash
+
+   kubectl create ns gmp-public
+
+2. Copy and apply the file from the `GKE documentation
+   <https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/nvidia-dcgm#install-exporter>`__
+   in the ``gmp-public`` namespace
+
+.. code:: bash
+
+   kubectl apply -n gmp-public -f /tmp/file.yaml
+
+3. Create a service for the DCGM exporter.
+
+.. code:: bash
+
+   kubectl apply -n gmp-public -f gkeDCGMExporterService.yaml
```

Comment:
This file is included in the Determined repository. Do we want to use the github file path here too?

Reply:
Never mind; I lost track ... earlier in the instructions it's all set up for that file to be in `pwd`: "_change directory to the determined repo ``tools/observability``._"